### PR TITLE
android touch issue

### PR DESCRIPTION
**Pull request by xianyinchen**
Handle touch event in SurfaceView, fix inconsistent touch area between Activity and View.

### CODE REVIEW

File: cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java [hunk 1]
```diff
@@ -45,4 +48,9 @@ public void run() {
             }
         });
     }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
+        return mTouchHandler.onTouchEvent(event);
+    }
```
Comment (minggo) [inline cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java:55]: 之前是需要判断引擎是否初始化好，现在不需要了？

Comment (wuzhiming) [reply cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java:55]: 需要

Comment (xianyinchen) [reply cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java:55]: 从安卓文档上是这样描述的
**surfaceCreated**
_This is called immediately after the surface is first created._
https://developer.android.com/reference/android/view/SurfaceHolder.Callback#surfaceCreated(android.view.SurfaceHolder)

**SurfaceView**
_The Surface will be created for you while the SurfaceView's window is visible; you should implement SurfaceHolder.Callback#surfaceCreated and SurfaceHolder.Callback#surfaceDestroyed to discover when the Surface is created and destroyed as the window is shown and hidden._
https://developer.android.com/reference/android/view/SurfaceHolder.Callback#surfaceCreated(android.view.SurfaceHolder)

Comment (wuzhiming) [reply cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java:55]: https://github.com/cocos-creator/engine-native/blob/41c6505343c7e91c6b8fc573949f1bdca90ce974/cocos/platform/android/java/src/com/cocos/lib/CocosActivity.java#L218

咱们 surfacecreate 之后还需要有个native的调用才算初始化成功

Comment (xianyinchen) [reply cocos/platform/android/java/src/com/cocos/lib/CocosSurfaceView.java:55]: 按照文档理解，初始化顺序应该是下面这样的，所以touch之前onSurfaceCreatedNative是调用了的
-> new SurfaceView  
-> new Surface
-> surfaceCreated
-> onSurfaceCreatedNative 
-> engineInit = true 
-> window = visible 
-> touch enable

**Review by minggo — APPROVED** [review]

**Review by wuzhiming — APPROVED** [review]

**Pull request merged**